docs/team/johndoe.adoc

```diff
@@ -9,7 +9,7 @@
 
 == Overview
 
-AddressBook - Level 4 is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+AddressBook - Level 4 is a desktop zone book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
```

Comment:
Hmm, did you replace all instances of address? No harm here though.

Reply:
oops didnt see that